src/monaco-adapter.ts

```diff
@@ -369,8 +369,8 @@ export class MonacoAdapter implements IEditorAdapter {
           []
         );
 
-        // Dispose Cursor Widget
-        this._cursorWidgetController.removeCursor(clientID);
+        // Dont remove the name tooltip like below, else it will cause flicker as you type
```

Comment:
Remove these?

Reply:
kept it on purpose, as a way to let other know to not do this in future.